service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillCreateAction.kt

```diff
@@ -194,38 +200,128 @@ class BackfillCreateAction @Inject constructor(
+                                  ) {
+                                    type = InputType.number
+                                    name = field
+                                    id = field
+                                    attributes["autocomplete"] = field
+                                  }
+                                }
+                              }
+                              div("sm:col-span-3") {
+                                val field = BackfillCreateField.RANGE_END.fieldId
+                                label("block text-sm/6 font-medium text-gray-900") {
+                                  htmlFor = field
+                                  +"""Range End (optional)"""
+                                }
+                                div("mt-2") {
+                                  input(
+                                    classes = "block w-full rounded-md bg-white px-3 py-1.5 text-base text-gray-900 outline outline-1 -outline-offset-1 outline-gray-300 placeholder:text-gray-400 focus:outline focus:outline-2 focus:-outline-offset-2 focus:outline-indigo-600 sm:text-sm/6",
+                                  ) {
+                                    type = InputType.number
+                                    name = field
+                                    id = field
+                                    attributes["autocomplete"] = field
+                                  }
+                                }
+                              }
+                            }
+                          }
                         }
                       }
                     }
-                    div("sm:col-span-3") {
-                      val field = BackfillCreateField.RANGE_END.fieldId
-                      label("block text-sm/6 font-medium text-gray-900") {
-                        htmlFor = field
-                        +"""Range End (optional)"""
+                  } else {
+                    // Range fields for new backfills (non-clone case)
+                    div("mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6") {
+                      div("sm:col-span-3") {
+                        val field = BackfillCreateField.RANGE_START.fieldId
+                        label("block text-sm/6 font-medium text-gray-900") {
+                          htmlFor = field
+                          +"""Range Start (optional)"""
+                        }
+                        div("mt-2") {
+                          input(classes = "block w-full rounded-md bg-white px-3 py-1.5 text-base text-gray-900 outline outline-1 -outline-offset-1 outline-gray-300 placeholder:text-gray-400 focus:outline focus:outline-2 focus:-outline-offset-2 focus:outline-indigo-600 sm:text-sm/6") {
```

Comment:
@adrw Are there good ways to handle all this copy and paste? I wonder if we should be doing something about this or not?